proto/README.md

```diff
@@ -24,6 +25,12 @@ zone | - | [ZoneEvent](sensr_proto/output.proto) | event list related to zone.
 losing | - | [LosingEvent](sensr_proto/output.proto) | event when SENSR lose tracking.
 health | - | [SystemHealth](sensr_proto/output.proto) | event when SENSR has trouble inside.
 
+## CustomMessage
+
+Field | Unit | Type | Description
+--- | --- | --- | ---
+field_of_regard | - | [PolygonBox](sensr_proto/type.proto) | List of negative field of regard which represent occlusion areas.
```

Comment:
The link to `sensr_proto/type.proto` gives a 404. And most others I tried too. I guess this doesn't work with the folder being a submodule, you might need to use the full link

Reply:
Thank you checking this. I replaced all the relative link to the direct link.